### ISSUE TITLE
add keybinding to accept the merge in merge editor

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#202877]
We need a default keybinding for `Complete Merge` in the merge editor and need to include it in the a11y help dialog

**Comment by isidorn** [comment]
What merge editor is this about? The simple one that shows >>>> decorations inline, or the merge editor with a blue button?
I could not figure this out from the screenshot.

Do we already have the required commands - is the feedback that there should be a default keybinding for them?

**Comment by meganrogge** [comment]
Yes, we already have the required commands. Default keybindings would help as would communicating to the user about which editor they've landed in

**Comment by meganrogge** [comment]
@jooyoungseo, I thought more about this. Is a user likely to open the editor help dialog to find these commands/keybindings? I worry they will not.

**Comment by jooyoungseo** [comment]
@meganrogge Isn't this the same issue and same question that can be applied to other contexts, such as copilot Chat? I think Alt+F1 needs to be their go-to reference until they get used to any new UIs.

**Comment by jooyoungseo** [comment]
Are you suggesting a separate verbosity/announcement to give a hint?

**Comment by meganrogge** [comment]
Maybe we should? I don't want it to be noisy though.
